tutorial/5. hypercube.ipynb

```diff
@@ -0,0 +1,931 @@
+       "2                  39.49340335509994     0.7893181165212338   \n",
+       "3                  39.49340335509994       4.14654462686305   \n",
+       "4                  39.49340335509994      9.168507779900754   \n",
+       "...                              ...                    ...   \n",
+       "6095               39.49340335509994 0.00027803476818283627   \n",
+       "6096               39.49340335509994  0.0004604018465094205   \n",
+       "6097               39.49340335509994  0.0009738277341721631   \n",
+       "6098               39.49340335509994  0.0008384210450644092   \n",
+       "6099               39.49340335509994  0.0009351807452481588   \n",
+       "\n",
+       "           CONC.B.mol m-3     CONC.C.mol m-3  \n",
+       "0      0.5582799990116556  7.505023274688886  \n",
+       "1      4.7041909379335936 2.8369484950718205  \n",
+       "2       7.692856761200801  5.479266737727308  \n",
+       "3      2.9064714790294497  9.718544736597146  \n",
+       "4      1.0452618789561743 2.3938272421305555  \n",
+       "...                   ...                ...  \n",
+       "6095 0.003847549196587605  16.46203270150234  \n",
+       "6096 0.005260220686453089 14.648738591753022  \n",
+       "6097 0.009950086229990268 20.221407581108544  \n",
+       "6098 0.007246355220872357  7.868645647524955  \n",
+       "6099 0.008232151867460548 7.7909351589915365  \n",
+       "\n",
+       "[6100 rows x 7 columns]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "concentrations_solved_expanded = []\n",
+    "time = []\n",
+    "for i in range(0, sim_length + 1, time_step_length):\n",
+    "    for j in range(0, num_grid_cells):\n",
+    "        concentrations_solved_expanded.append({key: value[j] for key, value in concentrations_solved[int(i/time_step_length)].items()})\n",
+    "        time.append(i)\n",
+    "df_expanded = pd.DataFrame(concentrations_solved_expanded)\n",
+    "df_expanded = df_expanded.rename(columns = {'A' : 'CONC.A.mol m-3', 'B' : 'CONC.B.mol m-3', 'C' : 'CONC.C.mol m-3'})\n",
+    "df_expanded['time.s'] = time\n",
+    "df_expanded['ENV.temperature.K'] = np.repeat(temperatures[0], (sim_length/time_step_length + 1.0) * num_grid_cells)\n",
+    "df_expanded['ENV.pressure.Pa'] = np.repeat(pressures[0], (sim_length/time_step_length + 1.0) * num_grid_cells)\n",
+    "df_expanded['ENV.air number density.mol m-3'] = np.repeat(state.get_conditions()['air_density'][0], (sim_length/time_step_length + 1.0) * num_grid_cells)\n",
+    "df_expanded = df_expanded[['time.s', 'ENV.temperature.K', 'ENV.pressure.Pa', 'ENV.air number density.mol m-3', 'CONC.A.mol m-3', 'CONC.B.mol m-3', 'CONC.C.mol m-3']]\n",
+    "display(df_expanded)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "05fb1f28",
+   "metadata": {},
+   "source": [
+    "## 6a. Visualizing the Results (Confidence Interval)\n",
```

Comment:
Typically I would expect to see a header `6` first, which describes some general information in this section. Then we can have `6a.` and `6b.` sub-header for different focuses.